recipe/meta.yaml

```diff
@@ -35,20 +35,29 @@ test:
     - repo_review.fixtures
     - repo_review.schema
   source_files:
+    - docs
     - tests
+    - README.md
+    - LICENSE
+    - noxfile.py
     - pyproject.toml
+    - .github
+    - .pre-commit-config.yaml
+    - .readthedocs.yaml
   requires:
     - rich-click
     - pip
     - pytest
-    # - sp-repo-review
+    - sp-repo-review
     - validate-pyproject >=0.14
   commands:
     - pip check
     - repo-review --help
     - validate-pyproject -E repo-review --help
-    # Remove tests that depends upon sp-repo-review
-    - rm tests/test_self.py && pytest
+    - mkdir src
+    - pytest -k "not test_cmd_dual"
+  downstreams:
```

Comment:
By the way, what benefit is gained from the downstreams field? I don't think I've used/seen that before, but I might have a few places I could use it. :)

Reply:
downstreams add testing for these packages. It allows identify issues with new revisions breaking other packages - I actually thought the original fails where due to something like that between sp-repo-review/repo-review

I am not sure how 'recommended' it is, it may add overhead to the whole system, but I feel that it'll help me avoid merging breaking changes - I am a newby within conda-forge